### ISSUE TITLE
Novacustom PiKVM lid problems

### ISSUE THREAD

**Issue by filipleple** [Dasharo/dasharo-issues#604]
### Device

Novacustom NVC41PZ

### Dasharo version

v1.7.1

### Brief summary

When running with the lid open, PiKVM is ignored by the BIOS and gets no display signal. When running with the lid shut, a couple minutes after booting into DTS the laptop goes into sleep mode, even though PiKVM is attached via HDMI as a monitor. Effectively, tests can't be fully run in either way.

### How reproducible

100%

### How to reproduce

Attach PiKVM to the laptop and try controlling the BIOS via PiKVM with the lid open - you will get no output. Then try booting into DTS with the lid shut - after a while the laptop will sleep and you'll have to reopen the lid to use it.

### Expected behavior

Ability to use BIOS via PiKVM with the lid open
and/or
Ability to use DTS with the lid closed

### Actual behavior

No BIOS output to PiKVM with the lid open, sleep mode in DTS with the lid shut

**Comment by miczyg1** [comment]
DTS may have configured to enter sleep while lid is shut. It is typically an OS setting

**Comment by macpijan** [comment]
@mkopec @TomaszAIR

**Comment by mkopec** [comment]
Yes I was just talking about this with @TomaszAIR , we just need to append `HandleLidSwitch=ignore` to `/etc/systemd/logind.conf`

Or keep the lid open and disconnect the eDP cable, for a hardware solution.

**Comment by macpijan** [comment]
Ubuntu does not suspend on the login screen with lid closed. We probably want DTS to behave the same.
For this iteration of testing, we may go with whatever suits you best, though.

**Comment by mkopec** [comment]
> Ubuntu does not suspend on the login screen with lid closed. We probably want DTS to behave the same.

I guess some distros inhibit suspend when an external monitor is attached, not sure if it's a systemd config option or some sort of service

**Comment by macpijan** [comment]
Yeah, in that case it probably comes from the GUI, like GNOME:
- https://www.reddit.com/r/gnome/comments/uthvg7/suspending_laptop_with_external_monitor_when_lid/
- https://github.com/GNOME/gnome-settings-daemon/blob/master/plugins/power/gsd-power-manager.c

For us in DTS me may simply disable handling of the lid switch entirely until we nee to use it.
